### PR TITLE
variable: add a varaible to read current timestamp

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@siddontang @shenli PTAL

File: session.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ type session struct {
 
 	store kv.Storage
 
-	// Used for test only.
+	// unlimitedRetryCount is ssed for test only.
```
Comment (siddontang) [inline session.go]: used

**Comment by siddontang** [comment]
Rest LGTM

**Comment by shenli** [comment]
LGTM

File: sessionctx/varsutil/varsutil.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ import (
 	"strings"
 	"time"
 
+	"fmt"
```
Comment (disksing) [inline sessionctx/varsutil/varsutil.go]: Not well formatted.

**Pull request merged**